express/properties/structural/inchi.py

```diff
@@ -0,0 +1,14 @@
+from express.properties.non_scalar import NonScalarProperty
```

Comment:
moved `inchi.py` and `inchi_key.py` below to new dir `express/properties/structural`